deeprank/learn/classMetrics.py

```diff
@@ -124,7 +124,14 @@ def mcc(yp, yt):
     fp = false_positive(yp, yt)
     fn = false_negative(yp, yt)
     tp, tn, fp, fn = map(np.float64, [tp, tn, fp, fn])
-    mcc = (tp * tn - fp * fn) / np.sqrt((tp + fp) * (tp + fn) * (tn + fp) * (tn + fn))
+
+    denominator = np.sqrt((tp + fp) * (tp + fn) * (tn + fp) * (tn + fn))
+    # if denominator is zero and causes an error, set it to 1 (source: https://en.wikipedia.org/wiki/Phi_coefficient) 
+    if denominator == 0:
+        mcc = (tp * tn - fp * fn) / 1
```

Comment:
from the link you refer to: 

> If any of the four sums in the denominator is zero, the denominator can be arbitrarily set to one; this results in a Matthews correlation coefficient of zero, which can be shown to be the correct limiting value. 

To make the code clearer, I would just set mcc = 0 instead of denominator = 1.
```suggestion
    denominator = np.sqrt((tp + fp) * (tp + fn) * (tn + fp) * (tn + fn))
    # if denominator is zero and causes an error, set it to 1 (source: https://en.wikipedia.org/wiki/Phi_coefficient) 
    if denominator == 0:
        mcc = 0
```